rocketpy/monte_carlo/mc_flight.py

```diff
@@ -0,0 +1,44 @@
+__author__ = "Mateus Stano Junqueira"
+__copyright__ = "Copyright 20XX, RocketPy Team"
+__license__ = "MIT"
+
+
+from typing import Any, Tuple, Union
+
+from pydantic import Field, StrictFloat, StrictInt
+
+from ..Flight import Flight
+from .DispersionModel import DispersionModel
+
+
+class McFlight(DispersionModel):
+    """Monte Carlo Flight class, used to validate the input parameters of the
+    flight to be used in the Dispersion class, based on the pydantic library. It
+    uses the DispersionModel class as a base class, see its documentation for
+    more information. The inputs defined here are the same as the ones defined
+    in the Flight class, see its documentation for more information.
+    """
+
+    flight: Flight = Field(..., exclude=True)
+    inclination: Any = 0
+    heading: Any = 0
+    initialSolution: Union[
+        Flight,
+        Tuple[
+            Union[StrictInt, StrictFloat],  # tInitial
```

Comment:
Does it means that, if we want, we can apply `initialSolution` inside the monte carlo loop?

Reply:
Yes! You can pass a initial solution to be used in all flight simulations of the dispersion analysis.

However the way it is currently done you can not add variations to these initial solutions, meaning you can not input a list of initial solutions. You can only input one initial solution that will be used for all flights....

Do you think we should allow the user to input a list of initial solutions?

Reply:
> Do you think we should allow the user to input a list of initial solutions?

I don't think so, that would be really weird to use. What would make more sense would be having an initial solution matrix composed by different parameters like (mean, std. dev, 'distr'). But that is not crucial here, no need to introduce that right now.

What we could do by now is to improve the docs to explain to the user that, if he wants to, he may use initial solution. But if I understood it right, if he sets an iniital solution to the Flight object, it will be automatically used by the McFlight object too.

Reply:
> if he sets an iniital solution to the Flight object, it will be automatically used by the McFlight object too.

No it won't. `initialSolution` does not go through the `@root_validator` so it won't check if there already is an initial solution on the flight object. This way the user would need to input the `initialSolution` in `McFlight`. 

Do you think we should change this? I hadn't even thought of this problem

It would be an easy change.

Reply:
> Do you think we should change this? I hadn't even thought of this problem

No, what we need for now is to release v1.0.0 ...

Please just add a simple "initialSolution option will be available in future releases" comment.
If there's demand, we solve the problem in the future, don't worry.